pkg/chart/v2/util/dependencies.go

```diff
@@ -48,10 +48,10 @@ func processDependencyConditions(reqs []*chart.Dependency, cvals Values, cpath s
 						r.Enabled = bv
 						break
 					}
-					log.Printf("Warning: Condition path '%s' for chart %s returned non-bool value", c, r.Name)
+					slog.Warn("Warning: Condition path '%s' for chart %s returned non-bool value", c, r.Name)
```

Comment:
```suggestion
					slog.Warn("Condition path '%s' for chart %s returned non-bool value", c, r.Name)
```

and so on.

Reply:
are you suggesting that i remove the "Warning:"

Reply:
That was my suggestion, yes.

Reply:
fixed this